src/render/view/ViewFlat.js

```diff
@@ -172,3 +172,45 @@ FORGE.ViewFlat.prototype.destroy = function()
     FORGE.ViewBase.prototype.destroy.call(this);
 };
 
+/**
+ * Get and set the repeat X beahvior.
+ * @name  FORGE.ViewFlat#repeatX
+ * @type {string}
+ */
+Object.defineProperty(FORGE.ViewFlat.prototype, "repeatX",
```

Comment:
Ouch, it's a bug: it's commited at start once for all for the moment. Please log a bug.